sys-libs/libcxx/libcxx-9999.ebuild

```diff
@@ -8,18 +8,22 @@ EAPI=6
 EGIT_REPO_URI="http://llvm.org/git/libcxx.git
 	https://github.com/llvm-mirror/libcxx.git"
 CMAKE_MIN_VERSION=3.4.3
+PYTHON_COMPAT=( python2_7 )
```

Comment:
k; i'm pretty sure i've seen ebuilds dying at global scope when no python_compat compatible version is enabled but you'd know better